tests/test_slots.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+#
+# A library that provides a Python interface to the Telegram Bot API
+# Copyright (C) 2015-2021
+# Leandro Toledo de Souza <devs@python-telegram-bot.org>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser Public License for more details.
+#
+# You should have received a copy of the GNU Lesser Public License
+# along with this program.  If not, see [http://www.gnu.org/licenses/].
+import abc
+import threading
+
+import pytest
+import inspect
+
+import telegram
+import telegram.ext
+from telegram.ext.utils import promise, webhookhandler
+from telegram.utils.helpers import DefaultValue
+from telegram.utils.request import Request
+
+
+@pytest.fixture(scope='function')
+def tg_classes():
+    tg_classes = inspect.getmembers(telegram, inspect.isclass)
+    tg_ext_classes = inspect.getmembers(telegram.ext, inspect.isclass)
+    utils_classes = [
```

Comment:
Can we somehow loop over the utils modules, too? adding the classes here manually smells like Forget-Me-Soon …

Reply:
ah yeah, I'll use the same logic I used in `test_filters.py`, so that'll cover all PTB classes too.